packages/@uppy/dashboard/types/index.d.ts

```diff
@@ -58,6 +58,7 @@ export interface DashboardOptions extends PluginOptions {
   onDragOver?: (event: DragEvent) => void
   onDragLeave?: (event: DragEvent) => void
   onDrop?: (event: DragEvent) => void
+  waitForThumbnailsBeforeUpload?: boolean,
```

Comment:
Should we make `DashboardOptions` and `ThumbnailGeneratorOptions` inherit those properties from a unique interface to avoid duplication? https://github.com/transloadit/uppy/blob/89f09c810a0b0374c2b72690f7adfe610492330e/packages/@uppy/thumbnail-generator/types/index.d.ts#L5-L12